diff --git a/tools/PCKRetrievalTool/App/linux/network_wrapper.cpp b/tools/PCKRetrievalTool/App/linux/network_wrapper.cpp
index cbd98e2..d151d68 100644
--- a/tools/PCKRetrievalTool/App/linux/network_wrapper.cpp
+++ b/tools/PCKRetrievalTool/App/linux/network_wrapper.cpp
@@ -393,7 +393,7 @@ network_post_error_t network_https_post(const uint8_t* raw_data, const uint32_t
         struct curl_slist *slist = NULL;
         slist = curl_slist_append(slist, "Content-Type: application/json");
 
-        if (user_token.empty()) {
+/*        if (user_token.empty()) {
             printf("\n Please input the pccs password, and use \"Enter key\" to end\n");
             int usless_ret = system("stty -echo");
             user_token = "user-token: ";
@@ -406,7 +406,7 @@ network_post_error_t network_https_post(const uint8_t* raw_data, const uint32_t
         } else {
             user_token = "user-token: " + user_token;
         }
-
+*/
         slist = curl_slist_append(slist, user_token.c_str());
         curl_easy_setopt(curl, CURLOPT_HTTPHEADER, slist);
         if (!g_use_secure_cert) {
